src/components/gridView/index.tsx

```diff
@@ -1,15 +1,16 @@
 import _ from 'lodash';
 import React from 'react';
 import {StyleSheet} from 'react-native';
-// TODO: we should use asBaseComponent here instead of using UIComponent directly
 import {Colors, Spacings} from 'style';
+// TODO: we should use asBaseComponent here instead of using UIComponent directly
```

Comment:
Don't you want to include this in your refactor?

Reply:
not at the moment..